doc/running.md

```diff
@@ -1,25 +1,31 @@
 # Building Pivid binaries
 
-It's currently best to build Pivid from source.
+It's currently best to build Pivid from source. 
 Please [open an issue](https://github.com/egnor/pivid/issues) if these
 instructions don't work!
 
+With a Raspberry Pi OS Lite installation, SD card size needs to be at least 16GB, otherwise the installation will fail.
+
 1. Use a Raspberry Pi (4B recommended) with a fully updated
 [bullseye](https://www.raspberrypi.com/news/raspberry-pi-os-debian-bullseye/)
 install and 2G+ RAM.
 
-2. Clone [this repository](https://github.com/egnor/pivid) (you will need
+2. Install some additional libraries `sudo apt update; sudo apt install pip python3-venv git git-lfs meson`
```

Comment:
`dev_setup.py` should do this already (and you shouldn't need the `meson` apt package, it pip installs it into the venv), so people shouldn't have to run it by hand? Did you find otherwise?

Reply:
My build broke when meson was not installed. I didn't further investigate, after installing meson, it just worked.

Reply:
Hmm. Leave this out for now, and let's debug what happens? I don't have the meson apt package installed and it works for me, so it ought to be possible. I'm trying to keep apt dependencies minimal (because it's hard to manage what version you get).

Reply:
I don't really care. I can remove this.

Reply:
Sounds good, and if you go through the install process again & run into the same problem, let me know and we'll figure it out

Reply:
Done